cli/dcos-spark/submit_builder.go

```diff
@@ -504,18 +504,15 @@ func buildSubmitJson(cmd *SparkCommand) (string, error) {
 		args.properties["spark.mesos.executor.docker.forcePullImage"] = "true"
 	}
 
-	// DCOS_SPACE
-	if cmd.submitDcosSpace == "" { // get the DCOS_SPACE from the marathon app
-		dispatcherId, err := getStringFromTree(responseJson, []string{"app", "id"})
-		if err != nil {
-			return "", err
-		}
-		cmd.submitDcosSpace = dispatcherId
+	// Get the DCOS_SPACE from the marathon app
+	dispatcherId, err := getStringFromTree(responseJson, []string{"app", "id"})
+	if err != nil {
+		return "", err
```

Comment:
Does it make sense to log the failure here?

Reply:
Is the `dispatcherId == ""` case something we should think of handling separately?